### ISSUE TITLE
Wash k8s plugin errors out when it encounters a context definition it doesn't recognize.

### ISSUE THREAD

**Issue by timidri** [puppetlabs/wash#163]
**Describe the bug**
Wash k8s plugin errors out when it encounters a context definition it doesn't recognize.

**To Reproduce**
1. Add a context definition to your kube config file like this:
```
- context:
    cluster: 127-0-0-1:8443
    user: developer/127-0-0-1:8443
  name: /127-0-0-1:8443/developer
```
2. start wash
3. See error
```
 ~/git/wash   master  ./wash server ~/wash_mnt
INFO[2019-03-22T11:57:30+01:00] Loading internal plugins
INFO[2019-03-22T11:57:30+01:00] Loading aws
INFO[2019-03-22T11:57:30+01:00] Loading docker
INFO[2019-03-22T11:57:30+01:00] Loading kubernetes
panic: plugin.NewEntry: received name myproject/127-0-0-1:8443/developer containing a /

goroutine 1 [running]:
github.com/puppetlabs/wash/plugin.NewEntry(0xc0005595f0, 0x22, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/dimitri.tischenko/git/wash/plugin/entryBase.go:51 +0x207
github.com/puppetlabs/wash/plugin/kubernetes.createContext(0x0, 0x0, 0x0, 0x0, 0x0, 0xc0004660c0, 0xc0004660f0, 0xc000466120, 0xc000466150, 0xc00053ef60, ...)
	/Users/dimitri.tischenko/git/wash/plugin/kubernetes/root.go:39 +0x1d7
github.com/puppetlabs/wash/plugin/kubernetes.(*Root).Init(0xc000550410, 0x1, 0xc0005039b0)
	/Users/dimitri.tischenko/git/wash/plugin/kubernetes/root.go:56 +0x3b3
github.com/puppetlabs/wash/plugin.(*Registry).RegisterPlugin(0xc000544900, 0x2855c40, 0xc000550410, 0x1, 0x1)
	/Users/dimitri.tischenko/git/wash/plugin/registry.go:37 +0x35
github.com/puppetlabs/wash/cmd.loadPlugin(0xc000544900, 0x263245a, 0xa, 0x2855c40, 0xc000550410)
	/Users/dimitri.tischenko/git/wash/cmd/server.go:174 +0xdf
github.com/puppetlabs/wash/cmd.loadInternalPlugins(0xc000544900)
	/Users/dimitri.tischenko/git/wash/cmd/server.go:184 +0x13c
github.com/puppetlabs/wash/cmd.serverMain(0xc00003db80, 0xc0005397d0, 0x1, 0x1, 0x0)
	/Users/dimitri.tischenko/git/wash/cmd/server.go:72 +0x1e1
github.com/puppetlabs/wash/cmd.toRunE.func1(0xc00003db80, 0xc0005397d0, 0x1, 0x1, 0x0, 0x0)
	/Users/dimitri.tischenko/git/wash/cmd/root.go:31 +0x4d
github.com/spf13/cobra.(*Command).execute(0xc00003db80, 0xc0005397b0, 0x1, 0x1, 0xc00003db80, 0xc0005397b0)
	/Users/dimitri.tischenko/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x473
github.com/spf13/cobra.(*Command).ExecuteC(0xc00003d400, 0xc00003d900, 0xc00015b680, 0xc00015b400)
	/Users/dimitri.tischenko/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0xc00003d400, 0x263b382, 0x12)
	/Users/dimitri.tischenko/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
github.com/puppetlabs/wash/cmd.Execute(0x0)
	/Users/dimitri.tischenko/git/wash/cmd/root.go:61 +0x2b
main.main()
	/Users/dimitri.tischenko/git/wash/wash.go:17 +0x60
```

**Expected behavior**
If the context definition is indeed incorrect, which I cannot really judge, I expect the k8s plugin to fail gracefully - just log this fact, skip the context and move on to the next one.

**Logs**
See above

**Environment (please complete the following information):**
 - OS: [macOS 10.14.3]
 - Version/Build number [dd8e20797f2d4418a26e8d9bbfd236fb1222c94c]
 - Cloud target [local k8s]

**Comment by ekinanp** [comment]
Yup, sounds like we may need to look at https://github.com/puppetlabs/wash/issues/156 after all

**Comment by timidri** [comment]
Update: the above problem happens whenever a context name contains a slash (/). So definitely something to fix.

**Comment by MikaelSmith** [comment]
Should be fixed with #156.
